CHANGELOG.md

```diff
@@ -25,6 +25,37 @@ playback to start before the entire session is downloaded and unpacked.
 Additionally, `tsh play` now supports a `--speed` flag for adjusting the
 playback speed.
 
+#### Teleport Operator can now be deployed against a remote Teleport cluster
+
+Prior to Teleport 15, the Teleport Kubernetes Operator had to run as a sidecar
+of the Teleport auth. It was not possible to use the operator in Teleport Cloud
+or against a Teleport cluster not deployed with the `teleport-cluster` Helm chart.
+
+In Teleport 15, the Teleport Operator can reconcile resource in any Teleport
+cluster. Teleport Cloud users can now use the operator to manage their resources.
+
+When deployed with the `teleport-cluster` chart, the operator now runs in a
+separate pod. This ensures that Teleport's availability won't be impacted if
+the operator becomes unready.
+
+See [the Standalone Operator guide](docs/pages/management/dynamic-resources/teleport-operator-standalone.mdx)
+for installation instructions.
+
+#### Teleport Operator now supports roles v6 and v7
+
+Starting with Teleport 15, newly supported kinds will contain the resource version.
+For example: `TeleportRoleV6` and `TeleportRoleV7` kinds will allow users to
+create Teleport Roles v6 and v7.
+
+Existing kinds will remain unchanged in Teleport 15, but will be renamed in
+Teleport 16 for consistency.
+
+To migrate an existing `TeleportRole` to a `TeleportRoleV7`, you must:
+- upgrade Teleport and the operator to v15
+- annotate the Custom Resource with `teleport.dev/keep: "true"`
+- delete the Custom Resource (it won't delete the resource thanks to the annotation)
```

Comment:
I think we should be explicit about the resource we are referring to.

```suggestion
- annotate the Role Custom Resource with `teleport.dev/keep: "true"`
- delete the Role Custom Resource (it won't delete the Teleport Role resource thanks to the annotation)
```

Reply:
This would make sense according to the Kubernetes nomenclature but I had to convert every "Role" and "User" to "role" and "user" in the operator docs during the review process. I will rename to `TeleportRole` to avoid ambiguity and be able to capitalize.